README.md

```diff
@@ -68,9 +68,9 @@ Unikraft supports the construction of multiple architectures, platforms, and ima
 
 ## Getting Started
 
-The fastest way to get started configuring, building and deploying Unikraft unikernels is to use our companion tool, [**kraft**][kraft].
+The fastest way to get started configuring, building and deploying Unikraft unikernels is to use our companion tool, [**kraftkit**][kraftkit].
 
-With kraft installed, you can download Unikraft components, configure your unikernel to your needs, build it and run it -- there's no need to be an expert!
+With `kraft` installed, you can download Unikraft components, configure your unikernel to your needs, build it and run it -- there's no need to be an expert!
```

Comment:
```suggestion
With `kraftkit` installed, you can download Unikraft components, configure your unikernel to your needs, build it and run it -- there's no need to be an expert!
```

Reply:
@RaduNichita I was thinking about this, but `kraftkit` installs `kraft` command. That's why I left it as it was. WDYT?

Reply:
Hi, in the README it should be clearer overall about the project name and what it provides. In the future we aim for the KraftKit project to install a number of binaries and tools which can be used, for example our containerd integration.

For now, I would simply rename the URL and update the text to something along the lines of:
```md
The fastest way to get started configuring, building and deploying Unikraft unikernels is to use the companion framework [KraftKit][kraftkit].

You can quickly and easily install KraftKit using the interactive installer. Simply run the following command to get started:

\```
curl --proto '=https' --tlsv1.2 -sSf https://get.kraftkit.sh | sh
\```

Alternatively, you can download the binaries from the [releases pages](https://github.com/unikraft/kraftkit/releases).
```

(Had to add `\` in front of the markdown block)